target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -434,6 +434,11 @@ define LegacyDevice/WPE72_16M
 endef
 LEGACY_DEVICES += WPE72_16M
 
+define LegacyDevice/WNR2000
```

Comment:
We have switched to a new image build code a while ago. The legacy*.mk files are only used for existing boards not yet converted to the new image build code.

Additions to this files (using the old image build code) are not accepted any longer. Please add your board to generic.mk and use the new image build code.